pkg/operator/oauth.go

```diff
@@ -96,14 +96,14 @@ func CreateOAuthClient(cr *v1alpha1.Console, rt *routev1.Route) (*oauthv1.OAuthC
 		logrus.Errorf("failed to create console oauth client : %v", err)
 		return nil, nil, err
 	} else {
-		logrus.Info("created console oauth client with secret ", randomBits)
+		logrus.Infof("created console oauth client with secret: %v...", randomBits[0:9])
```

Comment:
I'm worried someone might change the helper method to create a shorter secret at some point, and then we're accidentally logging the entire value. I think it's better to not log any part of the secret.

Reply:
Perhaps we have a `loglevel` toggle?  If they secrets mismatch, things don't work.  It would be a hassle to change source code for debugging.  I don't disagree, though it would probably be incorrect to shorten the secret.

Reply:
I don't think we should log any part of the secret regardless of log level.

They should always match here, right? Unless the create fails?

Reply:
they should always match, unless something goes funny.  Currently I think I always update them if anything is out of sync, w/ a new secret, every time.  So if someone meddled with one or the other they should re-sync.  

I'll nix it.